crates/nu-std/std/formats/mod.nu

```diff
@@ -28,3 +28,13 @@ export def "to ndjson" []: any -> string {
 export def "to jsonl" []: any -> string {
     each { to json --raw } | to text
 }
+
+# Convert from NDNUON, i.e. newline-delimited NUON, to structured
+export def "from ndnuon" []: [string -> any] {
+    lines | each { from nuon }
+}
+
+# Convert structured data to NDNUON, i.e. newline-delimited NUON
```

Comment:
```suggestion
# Convert structured data to NDNUON (newline-delimited NUON)
```

Reply:
fixed in https://github.com/nushell/nushell/pull/14334/commits/0e7e3c02f65f9b89c7bb631abcf46bc47099086e